src/lib.rs

```diff
@@ -316,6 +316,26 @@ impl Check {
             .deps(false)
             .silence(!config.is_verbose());
 
+        // If both the current and baseline rustdoc are given explicitely as a file path, we don't need to use the installed rustc, and this check can be skipped.
+        if !(matches!(self.current.source, RustdocSource::Rustdoc(_))
+            && matches!(self.baseline.source, RustdocSource::Rustdoc(_)))
+        {
+            let rustc_version_needed = rustc_version::Version::new(1, 65, 0);
```

Comment:
If we can lift this to a global static (ideally) or into the global config (next best), that'd be great.